ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/rlpx/framing/Framer.java

```diff
@@ -332,23 +333,24 @@ void frameMessage(final MessageData message, final ByteBuf buf) {
     buf.writeBytes(h).writeBytes(hMac);
 
     // Encrypt payload.
-    final byte[] f = new byte[frameSize + pad];
+    final MutableBytes f = MutableBytes.create(frameSize + pad);
```

Comment:
What do we gain by boxing this array into Mutable Bytes if we just unbox it at the end?  Since the framer is a hot spot I think the raw `byte[] f` should be preserved.

Reply:
The issue here is that Bytes doesn't support copyTo with a byte[] argument.

The MutableBytes object's `toArrayUnsafe` function returns the underlying byte array so the cost is relatively small?